infra/src/main/kotlin/fi/fta/geoviite/infra/publication/Publication.kt

```diff
@@ -67,6 +68,7 @@ data class ChangeValue<T>(
 data class PublicationChange<T>(
     val propKey: PropKey,
     val value: ChangeValue<T>,
+    // The string is intentionally nullable to allow omitting the whole field (change lists can be large)
     val remark: String?,
```

Comment:
Tämä ei varsinaisesti liity pullarin muutokseen, mutta vastaavissa voisi jälleen hyödyntää sitä ns. monta-eri-tyyppi-samassa-listassa -periaatetta, joista toisella olisi remark-kenttä non-nullablena ja toisella taas ei ollenkaan.

Reply:
Joo totta. Näitä ei myöskään tule sisäänpäin joten ei joudu tekemään mitään ihmeellisiä deserialisointihärveleitä sen takia. Voisin muuttaa tuon.